test/extended/storage/storageclass.go

```diff
@@ -70,7 +70,7 @@ var _ = g.Describe("[sig-storage][Feature:DisableStorageClass][Serial]", func()
 		sctest.SetAllowExpansion(false)
 
 		g.By("verifying the AllowVolumeExpansion reverts to true")
-		sctest.VerifyAllowExpansion(true)
+		sctest.VerifyAllowExpansion(true, func() {})
```

Comment:
this looks bad to me, can you pass `nil` and handle it inside `VerifyAllowExpansion`?

Reply:
Fixed